test/parallel/test-child-process-exec-error.js

```diff
@@ -35,10 +35,66 @@ function test(fn, code, expectPidType = 'number') {
 
 // With `shell: true`, expect pid (of the shell)
 if (common.isWindows) {
-  test(child_process.exec, 1, 'number'); // Exit code of cmd.exe
+  test(exec, 1, 'number'); // Exit code of cmd.exe
 } else {
-  test(child_process.exec, 127, 'number'); // Exit code of /bin/sh
+  test(exec, 127, 'number'); // Exit code of /bin/sh
 }
 
 // With `shell: false`, expect no pid
-test(child_process.execFile, 'ENOENT', 'undefined');
+test(execFile, 'ENOENT', 'undefined');
+
+
+// Verify that the exec() function throws when command parameter is not a valid string
+{
+  assert.throws(() => {
+    exec(123, common.mustNotCall());
+  }, {
+    code: 'ERR_INVALID_ARG_TYPE',
+    name: 'TypeError',
+    message: 'The "command" argument must be of type string. Received type number (123)'
```

Comment:
https://github.com/nodejs/node/blob/dbfcbe371cfedf33fe16856cc71b4ed06b0155ba/doc/contributing/writing-tests.md#L315-L325
```suggestion
    code: 'ERR_INVALID_ARG_TYPE',
```